products/qt.md

```diff
@@ -92,7 +89,6 @@ releases:
     extendedSupport: false
     latest: "5.13.2"
     latestReleaseDate: 2019-10-28
-    link: https://www.qt.io/blog/2019/06/19/qt-5-13-released
```

Comment:
I think this not a commercial version.

Reply:
What do you mean ? I just dropped the link because https://www.qt.io/blog/qt-5.13.2-released can be generated by the `changelogTemplate`.

Reply:
Sorry for the confusion, I was not aware of the template :smile: -> resolved